lib/linguist/vendor.yml

```diff
@@ -240,7 +240,7 @@
 - \.imageset/
 
 # Carthage
-- ^Carthage/
+- (^Carthage|\/Carthage)\/
```

Comment:
Could we use the shorter syntax `(^|/)Carthage/`?

Reply:
Good point, l'll change this.

Reply:
Changed, tests still passing.